src/views/components/header.html

```diff
@@ -29,6 +29,12 @@
 						<ul class="dropdown-menu">
 							<li><a href="/tools/debug">Debug tool</a></li>
 							<li><a href="/tools/purge">Purge cache</a></li>
+							<li role="separator" class="divider"></li>
```

Comment:
I actually meant this:
![image](https://user-images.githubusercontent.com/6192491/106932779-6ec1c200-6718-11eb-95fe-f05cd511cff8.png)

But your version, just without the X may be better:
![image](https://user-images.githubusercontent.com/6192491/106932867-8a2ccd00-6718-11eb-9121-11b2c4dea27e.png)


@jimaek wdyt?

Reply:
Looks cleaner but what about SEO? Do those page have good titles, descriptions and text?

Reply:
Yes, the pages themselves should be ok in that regard.